### PR TITLE
Review

### PR DESCRIPTION
**Pull request by jsoref**
This isn't a complete review, it's a checkpoint.

I actually wrote it on top of my grammar series, but I think it's helpful to push this forward.

### CODE REVIEW

File: draft-ietf-acme-acme.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ discussed on the ACME mailing list (acme@ietf.org).
 Certificates {{!RFC5280}} in the Web PKI are most commonly used to authenticate
 domain names.  Thus, certificate authorities in the Web PKI are trusted to
 verify that an applicant for a certificate legitimately represents the domain
-name(s) in the certificate.
+name(s) in a certificate.
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: This also doesn't make sense.

File: draft-ietf-acme-acme.md [hunk 2]
```diff
@@ -1801,7 +1802,7 @@ entity must both:
 
 {{security-considerations}} documents how the challenges defined in this
 document meet these requirements.  New challenges will need to document how they
-do.
+meet these requirements.
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: This phrasing is awkward and unnecessary.

File: draft-ietf-acme-acme.md [hunk 3]
```diff
@@ -994,7 +995,7 @@ Link: <https://example.com/acme/some-directory>;rel="index"
 
 If the client wishes to update this information in the future, it sends a POST
 request with updated information to the account URI.  The server MUST ignore any
-updates to the "key", or "order" fields or any other fields it does not
+updates to the "key" and "order" fields. The server MUST ignore, and any other fields it does not
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: This is unnecessary (and also broken grammar). Please revert.

File: draft-ietf-acme-acme.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ normative:
 Certificates in PKI using X.509 (PKIX) are used for a number of purposes,
 the most significant of which is the authentication of domain names.  Thus,
 certificate authorities in the Web PKI are trusted to verify that an applicant
-for a certificate legitimately represents the domain name(s) in the certificate.
+for a certificate legitimately represents the domain name(s) in a certificate.
```
Comment (jsoref) [inline draft-ietf-acme-acme.md]: how about "in certificates"?

There can't be one (`the`) certificate with multiple certificate authorities verifying.

File: draft-ietf-acme-acme.md (hunk 1 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: See previous.

File: draft-ietf-acme-acme.md (hunk 2 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: I had a lot of trouble figuring out what it meant.

Perhaps "New challenges will need similar documentation"?

I don't think that an implied "this" work.

File: draft-ietf-acme-acme.md (hunk 3 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: My concern w/ the original wording is that it I parse it as or(key, order, other fields)-[it does not know] -- but it knows key and order.

**Comment by jsha** [comment]
Closing this since most of the changes are small, and several of them are incorrect. I'll open a new PR porting in the grammer / punctuation fixes from the Account Key Roll-over section.

**Pull request closed without merging**